lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -23,12 +27,39 @@ class FirefoxDelegate {
       : 'none';
     this.firefoxConfig = options.firefox || {};
     this.options = options;
+    this.android = null;
```

Comment:
Minor: I don't think you need to initialise it at all since we always check if `this.android` exists before we use it, right?